aait/web/group-3/starter_project_group3/store/store.ts

```diff
@@ -0,0 +1,21 @@
+import { configureStore } from "@reduxjs/toolkit";
```

Comment:
It is better if this file is named index.ts than store.ts. it will seem clearer when we import it `import {store} from '@/store'` instead of  `import {store} from '@/store/store'`

Reply:
Fixed, changed to index.ts